reference/UiTPAS.v2.json

```diff
@@ -1413,15 +1445,17 @@
           },
           "price": {
             "type": "number"
+          },
+          "numberOfTickets": {
+            "type": "integer",
+            "minimum": 1,
+            "description": "Number of ticket sales that can be registered at this tariff. Default 1."
```

Comment:
Misschien moeten we deze bij de GET ook maar "required" maken? In de zin dat de API die altijd terug geeft, ook als het 1 is? Dan moet de client geen rekening houden met "default 1".

Reply:
OK.
Ik denk dat ik het net 'makkelijker' wou maken voor clients met die default, maar dat houdt geen steek.

Reply:
Voor alle duidelijkheid, ik bedoel enkel bij de `GET` he (het ophalen van de tarieven). Bij het registreren kunnen we het idd optioneel maken en default op 1 zetten.

Reply:
Ah maar ik zie dat je het al aangepast hebt en nu idd bij de `GET` required is maar bij de register niet. Top 👍